### ISSUE TITLE
phi3 output garbage on master

### ISSUE THREAD

**Issue by mmoskal** [EricLBuehler/mistral.rs#956]
## Describe the bug

On current master, phi3.5 outputs garbage. Sometimes it's somewhat coherent but typically not. It was working in 09d2e2596d9b0aa728749cf5614e40dd1a112b3b

```
[10:23 mistral.rs]$ ./target/release/mistralrs-server -n 1000  plain -m microsoft/Phi-3.5-mini-instruct -a phi3
2024-12-01T18:24:35.242220Z  INFO mistralrs_server: avx: false, neon: true, simd128: false, f16c: false
...
    Ex: `\system Always respond as a pirate.`
====================
> One line joke: 
to argintintargargegegegertargargerthegraphinargheatgrathheheheheheheheheheheehheahaxaxaxaxbinary binary
> 
```

## Latest commit or version

3d7e86ef316846acf6b098881bdaebe6fcf8c963

**Comment by EricLBuehler** [comment]
@mmoskal thanks for reporting this. I merged #958 which fixes it on my end, if you could confirm that would be great!

**Comment by mmoskal** [comment]
Thanks, seems to work now!
